backend/src/main/java/it/chalmers/gamma/bootstrap/BootstrapServiceHelper.java

```diff
@@ -0,0 +1,101 @@
+package it.chalmers.gamma.bootstrap;
+
+import it.chalmers.gamma.service.ApiKeyService;
+import it.chalmers.gamma.service.AuthorityLevelService;
+import it.chalmers.gamma.service.AuthorityService;
+import it.chalmers.gamma.service.FKITGroupService;
+import it.chalmers.gamma.service.FKITGroupToSuperGroupService;
+import it.chalmers.gamma.service.FKITSuperGroupService;
+import it.chalmers.gamma.service.ITClientService;
+import it.chalmers.gamma.service.ITUserService;
+import it.chalmers.gamma.service.MembershipService;
+import it.chalmers.gamma.service.PostService;
+
+import org.springframework.stereotype.Component;
+
+@SuppressWarnings({"PMD.ExcessiveParameterList"})
+@Component("hlp")
+public final class BootstrapServiceHelper {
```

Comment:
Seems like a bit of a bucket to hold it all, could it be split into more logical parts?

EntityBootstrapServices
AuthorityBootstrapServices
etc?

Now when bootstraping code is a bit more split maybe it's possible to depend only on the needed ones? :thinking:

Reply:
Due to time constraint Im not going to split it for now. But I will drop an issue to look into this.